dallinger/experiment_server/dashboard.py

```diff
@@ -599,3 +620,90 @@ def lifecycle():
     return render_template(
         "dashboard_cli.html", title="Experiment lifecycle Dashboard", **data
     )
+
+
+TABLE_DEFAULTS = {
+    "dom": "frtilpP",
+    "ordering": True,
+    "searching": True,
+    "select": True,
+    "paging": True,
+    "lengthChange": True,
+    "searchPanes": {"threshold": 0.99},
+}
+
+
+def prep_table_config(config):
```

Comment:
I know it's a small thing, but the name `config` has such strong associations with `Configuration` that my mind went there immediately. Since the name is only used a couple times, can we call it `experiment_table_config` or something?

Reply:
Maybe `datatable_config` (which is really what it is) or `database_table_config` or `database_datatable_config`?

Reply:
I settled on `datatables_options` since that's actually the best description of the object we are trying to construct here.